File: app/src/main/java/com/jqorz/planewar/base/IEntity.java
package com.jqorz.planewar.base;

import android.graphics.Bitmap;

/**
 *
 * @author j1997
 * @since 2020/4/7
 */
public interface IEntity {

    void setXY(int x,int y);

    int getX();

    void setX(int x);

    int getY();

    void setY(int y);

    Bitmap[] getBitmaps();
}

File: app/src/main/java/com/jqorz/planewar/base/IEntity.kt
package com.jqorz.planewar.base

import android.graphics.Bitmap

/**
 *
 * @author j1997
 * @since 2020/4/7
 */
interface IEntity {
    fun setXY(x: Int, y: Int)
    var x: Int
    var y: Int
    val bitmaps: Array<Bitmap>
}